app/controllers/events_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EventsController < ApplicationController
+  helper_method :current_user, :assist_event
+  before_action :user_authenticated
+  include EventsHelper
+
+  def index
+    @events = Event.select('events.*, users.id user_id, users.name creator_name').joins(:creator)
```

Comment:
The EventsController#index:

- [ ] lists past events, i.e. @past_events = Event.previous_events

- [ ] lists ucomping events, i.e. @upcoming_events = Event.upcoming_events